Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+// TODO: Audit
+public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
```

Comment:
This view is a bit different than all others because it's the titleView of navigationBar, right? I am still convinced that it should inherit our `_View` class. AKA `ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View`?

Reply:
@DominikBucher12 FYI this PR is just about moving the functionality, the audit will come in follow-up PRs